docs/reference/ml/anomaly-detection/ml-configuring-alerts.asciidoc

```diff
@@ -101,3 +172,160 @@ The name of an alert is always the same as the job ID of the associated
 {anomaly-job} on the page of the rule that lists the individual alerts. You can 
 open it via *{alerts-ui}* by selecting the rule name.
+
+
+[[action-variables]]
+== Action variables
+
+You can add different variables to your action. The following variables are 
+specific to the {ml} rule types.
+
+
+[[anomaly-alert-action-variables]]
+=== {anomaly-detect-cap} alert action variables
+
+Every {anomaly-detect} alert has the following action variables:
+
+`context`.`anomalyExplorerUrl`::
+URL to open in the Anomaly Explorer.
+
+`context`.`isInterim`::
+Indicates if top hits contain interim results.
+
+`context`.`jobIds`::
+List of job IDs that triggered the alert.
+
+`context`.`message`::
+A preconstructed message for the alert.
+
+`context`.`score`::
+Anomaly score at the time of the notification action.
+
+`context`.`timestamp`::
+The bucket timestamp of the anomaly.
+
+`context`.`timestampIso8601`::
+The bucket timestamp of the anomaly in ISO8601 format.
+
+`context`.`topInfluencers`::
+The list of top influencers.
+
+`context`.`topRecords`::
+The list of top records.
+
+
+[[anomaly-jobs-health-action-variables]]
+=== {anomaly-jobs-cap} health action variables
+
+Every health check has two main variables: `context.message` and 
+`context.results`. The properties of `context.results` may vary based on the 
+type of check. You can find the possible properties for all the checks below.
+
+==== _Datafeed is not started_ 
+
+`context.message`::
+A preconstructed message for the alert.
+
+`context.results`::
+Contains the following properties:
++
+.Properties of `context.results`
+[%collapsible%open]
+====
+`datafeed_id`:::
+The {dfeed} identifier.
+
+`datafeed_state`:::
+The state of the {dfeed}. It can be `starting`, `started`, 
+`stopping`, `stopped`.
+
+`job_id`:::
+The job identifier.
+
+`job_state`:::
+The state of the job. It can be `opening`, `opened`, `closing`, 
+`closed`, or `failed`.
+====
+
+==== _Model memory limit reached_
+
+`context.message`::
+A preconstructed message for the rule.
+
+`context.results`::
+Contains the following properties:
++
+.Properties of `context.results` 
+[%collapsible%open]
+====
+`job_id`:::
+The job identifier.
+
+`memory_status`:::
+The status of the mathematical model. It can have one of the following values:
+
+* `ok`: The models stayed below the configured value.
+* `soft_limit`: The model used more than 60% of the configured memory limit and 
+  older unused models will be pruned to free up space. In categorization jobs no 
+  further category examples will be stored.
+* `hard_limit`: The model used more space than the configured memory limit. As a 
+  result, not all incoming data was processed.
+
+`model_bytes_exceeded`:::
+The number of bytes over the high limit for memory usage at the last allocation 
+failure.
+
+`log_time`:::
+The timestamp of the model size statistics according to server time.
+
+`peak_model_bytes`:::
```

Comment:
my bad I missed the following fields in the tech design doc: `model_bytes` and `model_bytes_memory_limit`. Could you please add them?